apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/MoreActions/FinalDecisionDialog.tsx

```diff
@@ -0,0 +1,339 @@
+                        description: "Unsuccessful during assessment option",
+                        id: "wrqRvV",
+                      }),
+                    },
+                  ]}
+                  data-h2-margin-top="base(x1)"
+                />
+              )}
+              {finalAssessmentDecisionValue === "qualified" && (
+                <>
+                  <Heading
+                    data-h2-font-weight="base(700)"
+                    data-h2-margin-bottom="base(x.5)"
+                    data-h2-font-size="base(normal)"
+                  >
+                    {intl.formatMessage({
+                      defaultMessage: "Select an expiry date",
+                      id: "3MboNR",
+                      description:
+                        "Label for date selection input for expiry date",
+                    })}
+                  </Heading>
+                  <p>
+                    {intl.formatMessage({
+                      defaultMessage:
+                        "This is the amount of time this candidate will be considered for placement based on the results of this process. The usual amount of time is two years.",
+                      id: "DXjJyD",
+                      description:
+                        "Text describing expiry dates for candidates, what it is for and a recommendation",
+                    })}
+                  </p>
+                  <DateInput
+                    id="expiryDate"
+                    name="expiryDate"
+                    legend={intl.formatMessage({
+                      defaultMessage: "Select an expiry date",
+                      id: "3MboNR",
+                      description:
+                        "Label for date selection input for expiry date",
+                    })}
+                    rules={{
+                      required: intl.formatMessage(errorMessages.required),
+                      min: {
+                        value: strToFormDate(todayDate.toISOString()),
+                        message: intl.formatMessage(errorMessages.futureDate),
+                      },
+                    }}
+                    data-h2-margin-top="base(x.5)"
+                  />
+                </>
+              )}
+              <Well
+                color="warning"
+                fontSize="caption"
+                data-h2-margin-top="base(x1)"
+              >
+                <p
+                  data-h2-margin-bottom="base(x.5)"
+                  data-h2-font-weight="base(700)"
+                >
+                  {intl.formatMessage({
+                    defaultMessage: "Important",
+                    id: "IKGhHj",
+                    description: "Important note or caption",
+                  })}
+                </p>
+                <p>
+                  {intl.formatMessage({
+                    defaultMessage:
+                      "The candidate will be notified of any changes made in this form. ",
```

Comment:
Stray space at the end of this string.

Reply:
Done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/8978/commits/019d9ab14ea31a6398db07bd9cc0566e8860d8bd